SwiftMonkey/Monkey.swift

```diff
@@ -154,6 +154,15 @@ public class Monkey {
             actRegularly()
         }
     }
+    
+    /// Same as monkeyAround() but allows you to set a specific duration of time for the test
```

Comment:
Since we changed the signature we could also provide a more accurate Quick Help documentation, for example:
```
/// Generate random events or fixed-interval events based on a specific duration or infinitely.
///
/// - Parameter duration: The duration for which to generate the random events.
```